docs/changes.rst

```diff
@@ -97,6 +97,8 @@ Compatibility notes:
   - removed --save-space option (does not change behaviour)
 - removed --bypass-lock option
 - removed borg config command (only worked locally anyway)
+- compact command now requires access to the borg key if the repo is encrypted,
+  thus requiring a client to initiate compaction
```

Comment:
ah, good catch!

... is encrypted or authenticated.

and leave away the "client" part, people could also have the key on a server. "access to the borg key" is the important point.

Reply:
Done.

I just forgot to also delete that part from `compact`, the reason why I even suggested that :smile: Done too now.